docs/en/guides/Java-Plugin-Development-Guide.md

```diff
@@ -105,7 +105,7 @@ Create ExitSpan by operation name(e.g. service name, uri) and new **ContextCarri
 
     /**
      * Only use this method in explicit instrumentation, like opentracing-skywalking-bridge.
-     * It it higher recommend don't use this for performance consideration.
+     * It is highly recommended not to use this for performance consideration.
```

Comment:
Actually, this method has been removed in the 8.0.0. Please remove all these in the document, thanks.

Reply:
...Done